puppet/zulip/manifests/postgres_appdb_base.pp

```diff
@@ -90,7 +90,7 @@
 
     exec{"create_pgroonga_extension":
       require => File["$pgroonga_setup_sql_path"],
-      command  => "bash -c 'cat $pgroonga_setup_sql_path | psql -v ON_ERROR_STOP=1 zulip && touch $pgroonga_setup_sql_path.applied'",
+      command  => "bash -c 'cat $pgroonga_setup_sql_path | su postgres -c \"psql -v ON_ERROR_STOP=1 zulip\" && touch $pgroonga_setup_sql_path.applied'",
       creates  => "$pgroonga_setup_sql_path.applied",
```

Comment:
If we run as `postgres` user, we can't create "applied" file at '/usr/share/postgresql/VERSION/`. Should we change "applied" file path?